### ISSUE TITLE
MacVim crashes with VimBox enabled

### ISSUE THREAD

**Issue by Robert-M-Muench** [jordwalke/VimBox#23]
I have python 3.5 enabled but get this if I try to open a file or run:

:python3 import sys; print(sys.version) 

mac-pro:VimBox robby$ mvim
mac-pro:VimBox robby$ Fatal Python error: Py_Initialize: unable to load the file system codec
ImportError: No module named 'encodings'

Current thread 0x00007fff7474a000 (most recent call first):
Vim: Caught deadly signal ABRT
Vim: Finished.

**Comment by Robert-M-Muench** [comment]
The fix is to add this line to .vimrc when you use MacPorts Python installation.

let $PYTHONHOME="/opt/local/Library/Frameworks/Python.framework/Versions/Current"

Then things work.

**Comment by vdeep** [comment]
I'm having the same issue. I'm using python 3.6 with homebrew. I tried setting the PYTHONHOME path in vimrc.custom.before but still getting the same issue.

**Comment by jordwalke** [comment]
Yeah, I ran into this as well when trying python via brew. I'll reopen this and maybe someone can create a config that will work for all kinds of python installations.

**Comment by motiko** [comment]
Adding the following code to `vimrc.custom.before` solved this error for me. 

```
if has('python3')
    command! -nargs=1 Py py3 <args>
    set pythonthreedll=/usr/local/Frameworks/Python.framework/Versions/3.6/Python
    set pythonthreehome=/usr/local/Frameworks/Python.framework/Versions/3.6
else
    command! -nargs=1 Py py <args>
    set pythondll=/usr/local/Frameworks/Python.framework/Versions/2.7/Python
    set pythonhome=/usr/local/Frameworks/Python.framework/Versions/2.7
endif

```
